src/core/thread/tmf.cpp

```diff
@@ -123,6 +123,8 @@ bool Agent::HandleResource(const char *aUriPath, Message &aMessage, const Ip6::M
         Case(kUriEnergyScan, EnergyScanServer);
         Case(kUriActiveGet, MeshCoP::ActiveDatasetManager);
         Case(kUriPendingGet, MeshCoP::PendingDatasetManager);
+        Case(kUriAnnounceBegin, AnnounceBeginServer);
```

Comment:
Sending Announce messages is only for merging partitions, which can only be done by FTDs. I don't think MTDs need to implement this.